systemvm/js/ajaxviewer.js

```diff
@@ -184,7 +205,7 @@ KeyboardMapper.prototype = {
 			if(eventType == AjaxViewer.KEY_UP && (code == AjaxViewer.JS_KEY_ALT || code == AjaxViewer.JS_KEY_CTRL))
 				this.mappedInput.push({type : eventType, code: this.jsX11KeysymMap[code], modifiers: modifiers});
 			
-		} else if(eventType == AjaxViewer.KEY_PRESS && guestos == 'null') {
+		} else if(eventType == AjaxViewer.KEY_PRESS && guestos.toLowerCase() != 'windows') {
```

Comment:
This might be existing code, but for more cleaner way, do we want to remote all os strings and make them as constants or sort of enums defined at one place and use them with reference here? 
something like:
gustos.toLowerCase() != OsTypes.WINDOWS